.github/workflows/wheel_tests_and_release.yml

```diff
@@ -183,8 +174,53 @@ jobs:
         if: >
           startsWith(github.ref, 'refs/heads/v0.17') ||
           startsWith(github.ref, 'refs/tags/v0.17')
-
       - uses: actions/upload-artifact@v2
         with:
           name: wheels
           path: ./dist/*.whl
+
+
+  deploy:
+    name: Release
+    needs: [build_linux_37_and_above_wheels, build_macos_wheels, build_windows_wheels]
+    if: github.repository_owner == 'scikit-image' && startsWith(github.ref, 'refs/tags/v') && always()
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-python@v2
+        name: Install Python
+        with:
+          python-version: '3.7'
+      
+      - name: Install Dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install twine
+          pip install beautifulsoup4
+          pip install 'numpy>=1.16.5,!=1.18.0'
```

Comment:
Where does this set of dependencies come from?  Can we specify these in a requirements file?

Reply:
`beautifulsoup4` was intended to be a required dependency to install the previously used `wheel_uploader`, so it can be safely removed. The numpy requirements in there, seems to be related to an issue of incompatibility in the environment. But as it has already been fixed, we can remove that depency as well. I will test those removals and commit here once finished. Thanks @stefanv for pointing this out.

Reply:
HI @stefanv , I removed the bs4 dependency as it was needless, but the numpy restriction in there refers to the execution of `setup.py` in the next step. As it's removal causes 
```python
File "setup.py", line 14, in <module>
  from numpy.distutils.command.build_ext import build_ext
 ```
As we move forward we might need a separate requirements file for this, but right now I think that we can maintain this way to give more visibility. what do you think?

Reply:
I guess what had me confused is that this is available in any version of NumPy; so can we install NumPy from the skimage requirements file instead?

Reply:
Hint: `-r requirements/build.txt`

Reply:
Yes, that looks easiest. Thanks. Sorry for the delay.

Reply:
no delay. sometimes people implementing features are more in tune with challenges that arrise with what might seem like an "obvious" approach. I'm glad to know the suggestion works.